build_scripts/windows/resources/NOTICE.txt

```diff
@@ -0,0 +1,530 @@
+NOTICES AND INFORMATION
```

Comment:
we already have this file in src/promptlow/NOTICE.txt, could we reuse that file?